### ISSUE TITLE
dbf_to_dataframe can not process integer values?

### ISSUE THREAD

**Issue by gabriel-happle** [architecture-building-systems/CityEnergyAnalyst#1227]
the `dbf_to_dataframe` function for some reason converts `0` inputs to `nan` while `0.0` inputs are processed correctly.

**Comment by daren-thomas** [comment]
@gabriel-happle can you please provide a test-case? a minimal reproduction of the error?

**Comment by gabriel-happle** [comment]
@daren-thomas that error was spotted while processing the `void_deck` column an older version of the `architecture.dbf` file of the ecocampus with the `demand` script (or `data-helper`, I don't remember anymore) . 

I have to zip it to attach the file here:
[architecture.zip](https://github.com/architecture-building-systems/CityEnergyAnalyst/files/1994526/architecture.zip)

What is weird (and I'm realizing this just now), is that if I opened the file in ArcGIS or dbfeditor it showed `0.000000000000000` in that column. Just now I opened it with Excel and it showed nothing in that column (!). So maybe this is related to how ArcGIS displays empty cells in tables?

I fixed the file in the reference case repository by keying in `0.0` in every cell with ArcGIS.
